src/Api/Utils/ApiClient.php

```diff
@@ -1,8 +1,8 @@
 <?php
 /*
  * ApiClient.php
- * @author Martin Appelmann <hello@martin-appelmann.de>
- * @copyright 2021 Martin Appelmann
+ * @author Simon Schlossarek
+ * @copyright 2023 Simon Schlossarek
  */
 
```

Comment:
also here. please change it back. If you code a significant part of this project you will become a contributor.